### ISSUE TITLE
Add support for Chrome

### ISSUE THREAD

**Issue by giswqs** [UK-IPOP/scholar-network#1]
It seems the package only support Safari. Any plan for supporting Chrome?

![image](https://user-images.githubusercontent.com/5016453/144696256-6667cc3c-baaa-456e-95a2-59dad03321cf.png)

**Comment by nanthony007** [comment]
Hi @giswqs I just saw this issue. 

Supporting Chrome would be possible. 

I can probably add support to select your desired driver (Safari, Chrome, Firefox). This uses [selenium](https://selenium-python.readthedocs.io/index.html) under the hood so it would have to be one of the supported drivers. Additionally you will need to have the driver installed (which is separate process from installing the package itself).

Supported drivers: https://selenium-python.readthedocs.io/installation.html#drivers

**Comment by nanthony007** [comment]
This should be resolved with the newest version of the package. See the updated [examples](https://github.com/UK-IPOP/scholar-network#examples) for details.

TLDR: 

```python
>>>import scholar_network as sn
>>>sn.scrape_single_author(scholar_id='ZmwzVQUAAAAJ', scholar_name='Michelle Duong', driver='chrome')
```

Should now work 🙂 @giswqs

**Comment by giswqs** [comment]
That's awesome! Thanks a lot for implementing it.

**Comment by nanthony007** [comment]
No problem, let me know if there are issues. Thanks for requesting a great feature.

**Comment by giswqs** [comment]
It seems the parameter name is `preferred_browser`, not `driver`.

https://github.com/UK-IPOP/scholar-network/blob/3a188d0ef6a5fa11b4c78da8825cae6f19b4c8eb/src/scholar_network/scraping.py#L67

So the example should be:
```python
import scholar_network as sn
sn.scrape_single_author(scholar_id='ZmwzVQUAAAAJ', scholar_name='Michelle Duong', preferred_browser='chrome')
```

![image](https://user-images.githubusercontent.com/5016453/181133472-647c9732-4a5a-4fe8-9884-7ab409df82c6.png)

**Comment by nanthony007** [comment]
Adjusted. Thank you for catching that. I originally named it browser but renamed to avoid namespace collision.

**Comment by giswqs** [comment]
Great. Thanks. Another question: how do you visualize the network graph? I remembered I saw a relevant tutorial/blog post before, but I forgot where it came from. 

```python
g = sn.build_graph()
```

**Comment by nanthony007** [comment]
I use network and the graph edges... blog post coming tomorrow actually (I've been putting it off). I will share the notebook for the blog post tomorrow as well. 

Quickly now it's something like (if my memory serves):

```python
import networkx as nx
import scholar_network as sn

graph = sn.build_graph()
G = nx.Graph()
G.add_edges_from(graph.node_pairs())
pos = nx.spring_layout(G)
nx.draw(G, pos)
```

However you can also use plotly for something interactive -- ([example](https://plotly.com/python/network-graphs/)).

Both will be covered in the blog post tomorrow.

We use plotly on the COP scholar dashboard (cop-network-webapp) which you may be interested in. See the [repo](https://github.com/UK-IPOP/cop-network-webapp) or the [website](https://scholar-network-dashboard.herokuapp.com) which can be a little laggy so please be patient. 🙂

**Comment by giswqs** [comment]
The example throws an error. I look forward to reading your blog post. Thanks. 

![image](https://user-images.githubusercontent.com/5016453/181143382-3d7ff62d-204a-4d34-8dca-8ea3252e630c.png)

**Comment by nanthony007** [comment]
Sorry for the delay I ended up getting quite sick last week.  Article now published, available [here](https://nanthony007.medium.com/using-python-to-access-google-scholar-data-102127b8511a). @giswqs
